components/segment-bridge/development/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../base
```

Comment:
I don't think that the sandbox operator are installed by default in development. IIRC the user should specify he/she wants to install them using a flag.

Reply:
I see there's a script to run on a dev env, but I don't see anything that needs to be configured in those files, or any other component that does that. @MatousJobanek Any idea if there's anything I need to add for it to be configured?

Reply:
NVM my latest comment. 
While we configure the dev-env, we don't really need access to the sandbox operator as we install nothing. the only resource we really create there is the service account. I don't believe we need to change anything.

Reply:
@gbenhaim  you mean that the `toolchain-host-operator` namespace won't exist for the resources in the serviceaccount.yaml to be created in? Since we don't need the operator itself in development, and there is the issue I mentioned on my first comment, why not create a dummy namespace in development? unless I'm missing something about how to install it

Reply:
Fixed by adding a delete overlay